internal/util-logging/src/main/scala/sbt/util/LogExchange.scala

```diff
@@ -29,54 +28,18 @@ sealed abstract class LogExchange {
     Util.ignoreResult(configs.putIfAbsent(name, config))
   private[util] def removeConfig(name: String): Option[LoggerConfig] = Option(configs.remove(name))
 
-  @deprecated("Use LoggerContext to create loggers", "1.4.0")
   def logger(name: String): ManagedLogger = logger(name, None, None)
-  @deprecated("Use LoggerContext to create loggers", "1.4.0")
   def logger(name: String, channelName: Option[String], execId: Option[String]): ManagedLogger =
     LoggerContext.globalContext.logger(name, channelName, execId)
-  @deprecated("Use LoggerContext to unbind appenders", "1.4.0")
   def unbindLoggerAppenders(loggerName: String): Unit = {
     LoggerContext.globalContext.clearAppenders(loggerName)
   }
-  @deprecated("Use LoggerContext to bind appenders", "1.4.0")
-  def bindLoggerAppenders(
-      loggerName: String,
-      appenders: List[(XAppender, Level.Value)]
-  ): Unit = {
-    appenders.foreach {
-      case (a, l) =>
-        LoggerContext.globalContext
-          .addAppender(loggerName, new ConsoleAppenderFromLog4J(loggerName, a) -> l)
-    }
-  }
-  @deprecated("Use LoggerContext to bind appenders", "1.4.0")
   def bindLoggerAppenders(
       loggerName: String,
       appenders: Seq[(Appender, Level.Value)]
-  ): Unit = bindLoggerAppenders(loggerName, appenders.map { case (a, l) => a.toLog4J -> l }.toList)
-  @deprecated("unused", "1.4.0")
-  def loggerConfig(loggerName: String): LoggerConfig = configs.get(loggerName)
-
-  @deprecated("unused", "1.4.0")
-  lazy val asyncStdout = buildAsyncStdout
-  @deprecated("unused", "1.4.0")
-  private[sbt] def buildAsyncStdout: AsyncAppender = {
-    val ctx = XLogManager.getContext(false) match { case x: XLoggerContext => x }
-    val config = ctx.getConfiguration
-    val appender = ConsoleAppender("Stdout").toLog4J
-    // CustomConsoleAppender.createAppender("Stdout", layout, null, null)
-    appender.start
-    config.addAppender(appender)
-    val asyncAppender: AsyncAppender = AsyncAppender
-      .newBuilder()
-      .setName("AsyncStdout")
-      .setAppenderRefs(Array(AppenderRef.createAppenderRef("Stdout", XLevel.DEBUG, null)))
-      .setBlocking(false)
-      .setConfiguration(config)
-      .build
-    asyncAppender.start
-    config.addAppender(asyncAppender)
-    asyncAppender
+  ): Unit = {
+    appenders.map(LoggerContext.globalContext.addAppender(loggerName, _))
+    ()
   }
```

Comment:
Removes the deprecation warnings on the `logger`, `unbindLoggerAppenders` and `bindLoggerAppenders` methods given https://github.com/sbt/sbt/issues/6734.